src/components/claiming/UploadFile.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <div>
+    <NavigationCard
+      data-test="header-2"
+      class="wallet-disclaimer"
+      title="Import claiming file"
+      :previousStep="previousStep"
+      :nextStep="nextStep"
+      previousText="Back"
+      nextText="Next"
+    >
+      <InformativeContent :subtitle="subtitle" :texts="texts" />
+      <el-upload
+        class="upload-file"
+        ref="upload"
+        accept="application/json"
+        :auto-upload="false"
+        :multiple="false"
+        :http-request="handleUpload"
+        :on-change="handleUpload"
+        action=""
+        :limit="1"
+        list-type="text"
+        :on-exceed="handleExceed"
+      >
+        <el-button size="small" type="primary">Click to upload file</el-button>
+        <div slot="tip" class="el-upload__tip">Only application/json files supported</div>
+      </el-upload>
+    </NavigationCard>
```

Comment:
When you click in the `next` button and you haven't uploaded the file, you should get an error below the file button with a text like: `You have to upload the file to continue`. On the other hand, the next button should be disabled and when you hover/click it, a tooltip should appear saying that you have to upload the file.

Can you create an issue and add a comment here linking it?